cargoTracking.md

```diff
@@ -0,0 +1,123 @@
+
+### Versiyon Bilgisi
+
+_Versiyon_ : v1
+
+### URI şeması
+
+_Host_ : api.navlungo.com
+_Host(Test)_ : api-qa.navlungo.com
+_Schemes_ : HTTPS
+
+### Kabul Edilen Girdi Formatları
+
+- `application/json`
+
+### Üretilen Çıktı Formatları
+
+- `application/json`
+
+### Yetkilendirme
+
+Bu api Oauth2 **client_credentials** akışı ile oluşturulan token'lar ile çağrılabilir. Token alırken scope parametresine **public_cargo_tracking** değeri gönderilmelidir.
+
+### Operasyonlar
+
+- [getCargoTrackingInfo](#getCargoTrackingInfo)<br>
+
+<a name="paths"></a>
+
+## Paths
+
+<a name="getCargoTrackingInfo"></a>
+
+### GET tracking/v1/cargo
+
+**Operasyon: getCargoTrackingInfo**
+
+#### Açıklama
+
+Express taşımalar için kargo takip bilgilerini döndürür.
+
+#### Parametreler
+
+| Tip             | İsim                                 | Açıklama                    | Şema                         |
+| --------------- | ------------------------------------ | --------------------------- | ---------------------------- |
+| **Querystring** | **courier** <br>_zorunlu_            | Taşıyıcı firma kodu         | < [Courier](#courier) > Enum |
+| **Querystring** | **trackingNumber** <br>_zorunlu_     | Takip numarası              | -                            |
+| **Querystring** | **originCountry** <br>_zorunlu_      | Kargonun çıkış yaptığı ülke | -                            |
+| **Querystring** | **destinationCountry** <br>_zorunlu_ | Teslimat ülkesi             | -                            |
+
+#### Yanıtlar
+
+| HTTP Kodu | Açıklama                                                        | Şema                                  |
+| --------- | --------------------------------------------------------------- | ------------------------------------- |
+| **200**   | Başarılı                                                        | [TrackingResponse](#trackingResponse) |
+| **400**   | İstek doğrulamasında hata oluştu veya istek geçersiz.           | [Error](#error)                       |
+| **401**   | Yetkilendirme hatası. Access token geçersiz veya süresi dolmuş. | [Error](#error)                       |
+| **500**   | İstek sırasında beklenmedik bir hata oluştu.                    | [Error](#error)                       |
+
+<a name="definitions"></a>
+
+####Tanımlar
+
+<a name="courier"></a>
+
+| Kod          | Taşıyıcı |
+| ------------ | -------- |
+| **dhl**      | DHL      |
+| **fedex**    | FedEx    |
+| **tnt**      | TNT      |
+| **ups**      | UPS      |
+| **dpd**      | DPD      |
+| **gls**      | GLS      |
+| **aramex**   | ARAMEX   |
+| **hermes**   | HERMES   |
+| **hellmann** | HELLMAN  |
+
+<a name="trackingResponse"></a>
+
+**TrackingResponse**
+
+| Ad                               | Açıklama                      | Şema                                       |
```

Comment:
```suggestion
| Ad                               | Her Zaman Mevcut |Şema                      | Açıklama                                       |
```
Dönüş tipinde zorunlu kelimesi tuhaf geldi. Onun yerine Her Zaman Mevcut kolonu kullanabiliriz. En uzun içerikte açıklamada olacağı için sona eklemeyi tercih edebiliriz. Böylece bütün alanlar rahatlıkla okunabilir.

Reply:
Ad yerine de Parametre Adı diyebiliriz

Reply:
danke